icubam/backoffice/templates/map.html

```diff
@@ -22,7 +22,7 @@
   let showed = new Set()
   let all_showed = false
   function initMap () {
-    plotMap(data, center)
+    plotMap(data, center, "{{ _('Tout montrer') }}", "{{ _('Clicker pour afficher') }}")
```

Comment:
nice, but i would put those in English here and add two entries in fr.csv. This would be more consistent IMO